modules/default.nix

```
@@ -374,7 +390,7 @@ in
     ];
 
     systemd.services.nvidia-cdi-generate = {
```

```suggestion
    systemd.services.nvidia-cdi-generate = mkIf cfg.container-toolkit.enable {
```

I would prefer not creating the systemd unit at all if our container-toolkit option is not enabled. The behavior of nixpkgs when `systemd.services.<name>.enable` is false is to create a masked unit, which means the unit still exists on the running system. I think not having the unit exist at all is better. `systemd.services.<name>.enable` defaults to `true`, so we can omit setting it at all.